mne/__init__.py

```diff
@@ -65,7 +65,9 @@
                            write_source_spaces, setup_source_space,
                            setup_volume_source_space, SourceSpaces,
                            add_source_space_distances, morph_source_spaces,
-                           get_volume_labels_from_aseg)
+                           get_volume_labels_from_aseg,
+                           get_volume_labels_from_aseg_AP,
```

Comment:
get_volume_labels_from_aseg_AP removed
I'm testing the new changes and after I'll push the modified files